decidim-core/app/packs/src/decidim/redesigned_heading_tag.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Ensures the aside heading tag is transformed to h2 or h1 if the drawer is shown or hidden
```

Comment:
I didn't try it out, but how about the other headings? Wouldn't this introduce [illogical heading order](https://docs.decidim.org/en/develop/develop/guide_accessibility.html#_illogical_heading_order)?

Reply:
Just to be clear, I'm referring to this part of our A11Y docs: 

`Every page should have a logical heading order when using the <h1>, <h2>, <h3>, <h4>, <h5> and <h6> heading elements. This means that you cannot add e.g. a <h3> if the previous heading was <h1> before the sub-level heading element appears in the document. <h1> should be followed by <h2>. Under the <h2>, you can add <h3>.`

This is the current structure of an opened drawer page according to WAVE: 

![Selection_441](https://user-images.githubusercontent.com/717367/204247154-f381ef94-b90d-43d0-9c62-d3cc693d5eb4.png)

Ignoring header and footer, as that needs to be fixed on other PRs, what I see is: 

h2. Accountability
h2. Global execution status
h1. Animi provident (the project page) 
h3. 2 comments

I understand that this should be: 

h1. Animi provident (the project page) 
h2. 2 comments

As the result title should be the title of the page and is the h1.

Reply:
I'm moving this to https://github.com/decidim/decidim/issues/10122. It's being overcomplicated to handle the situations we're facing with this matter, if you guys have something in mind to help this out, we'll appreciate it.

In the meanwhile, we can go ahead.